src/dtos/user-dto.ts

```diff
@@ -0,0 +1,19 @@
+import { User } from '../entity/user';
+
+export class UserDto {
+    id: string;
+    oidcId: string;
+    provider: string;
+    name: string;
+    email: string;
```

Comment:
In the general case this can change at every login. We need to decide if we care about a history of changes or if there is any user facing thing we need to do when it happens.

A change here may indicate a change or it may indicate a reassignment. We should understand our persistent ID so that we can make a change here an entirely user facing issue.

What do we need this for and therefore what do we do when/if we don't get it for a particular user?